app/views/dashboards/subscriptions.html.erb

```diff
@@ -0,0 +1,39 @@
+<% title "Subscriptions #{@source.title}" %>
+
+<div class="crayons-layout crayons-layout--1-col">
+  <header class="flex items-center justify-between p-2 m:p-0">
+    <div>
+      <h1 class="fs-2xl s:fs-3xl">Subscriptions</h1>
+      <h2 class="fs-base s:fs-xl fw-medium">
+        <em class="fs-italic fw-normal color-base-60">for</em>
+        <a href=<%= @source.path %>><%= @source.title %></a>
+      </h2>
+    </div>
+    <%#TODO: [@thepracticaldev/delightful]: uncomment this when ready to implement CSV exports%>
+    <%#<button type="button" class="crayons-btn crayons-btn--secondary">Export CSV</button>%>
+  </header>
+
+  <main>
+    <% if @subscriptions.any? %>
+      <table class="crayons-table w-100">
+        <tbody class="crayons-card">
+          <% @subscriptions.each do |subscription| %>
+            <% subscriber = subscription.subscriber %>
+            <tr>
+              <td class="fw-medium"><a href=<%= "/#{subscriber.username}" %> <%= "#{subscriber.name} DEV profile" %>><%= subscriber.name %></a></td>
```

Comment:
is it possible to user a path helper here instead of a direct `"/#{subscriber.username}"` href?

Reply:
It is! I created a path helper for this path and used that instead.